src/components/expander.js

```diff
@@ -42,4 +42,4 @@ var Expander = React.createClass({
   }
 });
 
-module.exports = Expander;
\ No newline at end of file
+export default Expander;
```

Comment:
Just curious, why not:   `export default const Expander = React.createClass(...);` ? Do you prefer this way for a reason, or just because the code was like this already?